src/crypto/CryptState.h

```diff
@@ -13,6 +13,15 @@ class CryptState {
 private:
 	Q_DISABLE_COPY(CryptState)
 public:
+	static const unsigned int ivLength   = 0;
+	static const unsigned int keyLength  = 0;
```

Comment:
IV means `initial vector` and this is a well-known cryptographic norm (including this abbreviation). (Sorry to pretend to be a cryptographic insider again but this is actually something that appears in all literature I have read)

Reply:
Okay then the abbreviation is fine. Especially since I would not have been able to understand more if it was called `initial vector` :sweat_smile: 

> Sorry to pretend to be a cryptographic insider again but this is actually something that appears in all literature I have read

No need to apologize. It is a fact that you know more cryptography stuff than I do :shrug: